lib/core/test_suite.js

```diff
@@ -117,8 +117,12 @@ export class TestSuite {
     return this.tests().filter(test => test.isSkipped()).length;
   }
 
+  explicitlySkippedCount() {
```

Comment:
For simplicity, let's count the two variants of skip (the fail fast and this new one) as "skipped", because otherwise we need to change the console UI to handle this new counter.

With this behaviour, counters do not add up when you skip tests. For a suite with 4 tests (1 error, 1 fail, 1 success and 1 skip) the output is the following:

```
================================================================================

Total tests:
4 test(s), 1 passed, 1 failed, 1 error(s)
================================================================================
```